packages/vuetify/src/components/VLayout/VLayoutItem.tsx

```diff
@@ -1,34 +1,46 @@
+// Styles
+import './VLayoutItem.sass'
+
 // Composables
 import { makeLayoutItemProps, useLayoutItem } from '@/composables/layout'
 
 // Utilities
-import { computed, defineComponent, toRef } from 'vue'
+import { defineComponent, toRef } from 'vue'
 import { makeProps } from '@/util'
 
 // Types
-import type { Prop } from 'vue'
+import type { PropType } from 'vue'
 
 export default defineComponent({
   name: 'VLayoutItem',
 
   props: makeProps({
     position: {
-      type: String,
+      type: String as PropType<'top' | 'right' | 'bottom' | 'left'>,
       required: true,
-    } as Prop<'top' | 'right' | 'bottom' | 'left'>,
+    },
+    size: {
+      type: [Number, String],
+      default: 300,
+    },
+    modelValue: Boolean,
     ...makeLayoutItemProps(),
   }),
 
   setup (props, { slots }) {
     const styles = useLayoutItem(
       props.name,
       toRef(props, 'priority'),
-      computed(() => props.position ?? 'left'),
-      toRef(props, 'size')
+      toRef(props, 'position'),
+      toRef(props, 'size'),
+      toRef(props, 'size'),
+      toRef(props, 'modelValue')
     )
 
     return () => (
-      <div style={ styles.value } v-slots={ slots } />
+      <div class="v-layout-item" style={ styles.value }>
```

Comment:
don't render element if no default slot exists.

Reply:
Not sure what you mean here?

Reply:
```tsx
return () => (
  { slots.default && (
	<div class="v-layout-item" style="{ styles.value }>
      { slots.default() }
    </div>
  ) }
)

Reply:
That doesn't apply here because the div is the root element of the component.

Reply:
Whoops lol